language-support/ts/daml-react/hooks.ts

```diff
@@ -138,14 +138,14 @@ export function useFetchByKey<T extends object, K, I extends string>(template: T
  * @typeparam I The template id type.
  *
  * @param template The template of the contracts to match.
- * @param queryFactory A function returning a query.
+ * @param queryFactory A function returning a query. If the query is omitted, all visible contracts of the given template are returned.
  * @param queryDeps The dependencies of the query (for which a change triggers an update of the result)
  *
  * @return The matching contracts.
  *
  */
-export function useStreamQuery<T extends object, K, I extends string>(template: Template<T, K, I>): QueryResult<T, K, I>
 export function useStreamQuery<T extends object, K, I extends string>(template: Template<T, K, I>, queryFactory: () => Query<T>, queryDeps: readonly unknown[]): QueryResult<T, K, I>
+export function useStreamQuery<T extends object, K, I extends string>(template: Template<T, K, I>): QueryResult<T, K, I>
```

Comment:
@hurryabit I changed the order of these signatures so that the documentation is created for the version with most parameters. I hope this doesn't change anything in semantics.

Reply:
Since the signatures don't overlap, this should be fine. It would also only affect type checking and nothing at runtime. Can you please do the same for `useQuery` as well if necessary?

Reply:
for `useQuery` it's already right.